initfiles/sbin/backup.sh.in

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+# Expects 
+# 1:    target cluster
+# 2:    logfile
+source @INSTALL_DIR@/etc/init.d/hpcc_common
+set_environmentvars
+source @ADMIN_PATH@/hpcc_setenv ""
+
+component=$(basename ${PWD})
+target="$1"
+logfile="$2"
```

Comment:
Should there be some checking here that right num params has been passed etc. and log usage if not?